invenio_cli/cli.py

```diff
@@ -147,7 +147,12 @@ def update(install_js):
 def destroy(local, verbose):
     """Removes all associated resources (containers, images, volumes)."""
     cli_config = CLIConfig()
-    commands = LocalCommands(cli_config)
+
+    if local:
+        commands = LocalCommands(cli_config)
+    else:
+        commands = Commands(cli_config, local)
```

Comment:
`Commands` is a factory of "sub-commands" classes (i.e. `LocalCommands` and `ContainerizeCommands`. The *if-else* might not be needed ;).

Check how the `Commands` constructor works, and how the `demo` command is implemented. This, is a good first step :), but it can be much simpler by properly using the factory.